doc/Glossary/index.md

```diff
@@ -5,74 +5,198 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
A particular way of rebalancing the workload between the [Operations services]({{root_url}}Glossary/#operations-service) within a [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) to achieve the desired load for each service.